presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -823,6 +823,22 @@ private void assertSTGeometryN(String wkt, int index, String expected)
         assertFunction("ST_ASText(ST_GeometryN(ST_GeometryFromText('" + wkt + "')," + index + "))", VARCHAR, expected);
     }
 
+    @Test
+    public void testSTGeometries()
+    {
+        assertSTGeometries("POINT empty", null);
+        assertSTGeometries("POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))", ImmutableList.of("POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))"));
```

Comment:
Let's test all geometry types. E.g. point, multi-point, linestring, multi-linestring, polygon, multi-polygon, geometry collection.

For brevity, let's use varags in `assertSTGeometries` (see https://docs.oracle.com/javase/8/docs/technotes/guides/language/varargs.html)

```
private void assertSTGeometries(String wkt, String... expected)


assertSTGeometries("POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))", "POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))");
assertSTGeometries("MULTIPOINT (1 2, 4 8, 16 32)", "POINT (1 2)", "POINT (4 8)", "POINT (16 32)");
```